pkgs/os-specific/linux/system76-acpi-dkms/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, fetchpatch, kernel }:
+let
+  version = "1.0.1";
+  sha256 = "0jmm9h607f7k20yassm6af9mh5l00yih5248wwv4i05bd68yw3p5";
+in
+stdenv.mkDerivation {
+  pname = "system76-acpi-dkms";
+  inherit version;
+
+  src = fetchFromGitHub {
+      owner = "pop-os";
+      repo = "system76-acpi-dkms";
+      rev = version;
+      inherit sha256;
+    };
+
+  hardeningDisable = [ "pic" ];
+  dontStrip = true;
```

Comment:
Question: What does the `dontStrip = true;` do here? Does this use debug symbols in any form? Same for the other kernel modules

Reply:
Nope.  Removed.